fideslib/models/client.py

```diff
@@ -118,14 +118,19 @@ def credentials_valid(self, provided_secret: str, encoding: str = "UTF-8") -> bo
 
 
 def _get_root_client_detail(
-    root_client_id: str, root_client_hash: tuple | None = None, encoding: str = "UTF-8"
+    config: FidesConfig,
+    scopes: list[str] | None = None,
```

Comment:
Should you change the type since scopes are now required?

Reply:
Good point. I'll change this.